docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,476 @@
+- `Goldstein et al., 2014 <https://esurf.copernicus.org/articles/2/67/2014/esurf-2-67-2014-discussion.html>`_
+
+The process-based approach solves the suspended sediment transport by the three-dimensional advection-diffusion equation
+(i.e., the scaler transport equation),  with an additional source/sink term for sediment vertical settling and exchange
+with the bed.
+
+Suspended sediment transport requires the knowledge of the sediment settling velocity.
+Sediment settling velocity is a function of the sediment geometry (e.g., particle size, shape and roundness),
+the fluid characteristics (density and viscosity), and well as the fluid turbulence velocity.
+Two sediment settling veloity formulations are selected to implement into MPAS-Ocean, including
+
+- `Soulsby, 1997 <https://www.icevirtuallibrary.com/doi/abs/10.1680/doms.25844.fm>`_
+- `Cheng, 1997 <https://doi.org/10.1061/(ASCE)0733-9429(1997)123:2(149)>`_
+
+
+Requirement: Morphological change
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/20
+
+Contributors: Zhendong Cao
+
+Morphological change here stands for the change of sea floor elevation (bathymetry) due to the convergence/divergence of net
+sediment flux during a time period. The Exner equation (`Paola and Voller, 2005 <https://doi.org/10.1029/2004JF000274>`_)
+is implemented to compute the morphological change.
+
+
+Algorithm Design
+---------------------------
+
+Algorithm Design: Sediment classes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/16
+
+Contributors: Zhendong Cao
+
+The number of sediment classes will be defined manually and considered as a dimension of sediment-related variables.
+
+Algorithm Design: Bedload sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/16
+
+Contributors: Zhendong Cao
+
+In practice, a bedload coefficient should be defined to adjust the bedlad transport rate (see `COAWST code here
+<https://github.com/jcwarner-usgs/COAWST/blob/master/ROMS/Nonlinear/Sediment/sed_bedload.F#L608-L612>`_).
+
+The Meyer-Peter-Mueller and Soulsby-Dammgaard share the same expression of the bedload transport rate
+
+.. math::
+    q_b = \Phi \sqrt{(s-1)g D_{50}^3}\rho_s
+
+in which :math:`q_b` is the bedload transport rate (:math:`kg m^{-1}s^{-1}` ), :math:`\Phi` is dimensionless bedload
+transport rate, :math:`s=\rho_s-\rho_w` is the sediment specific density in water, :math:`\rho_s` and :math:`\rho_w`
+are the sediment denisty and the water density, respectively (:math:`kg m^{-3}`), :math:`g` is the gravitational
+acceleration (:math:`m s^{-2}`), :math:`D_{50}` is the median grain diameter (:math:`m`).
+
+The main difference of the two formulations is the computation of :math:`\Phi`, which is a function of another
+dimensionless parameter called Shields parameter :math:`\theta`
+
+.. math::
+	\theta = \frac{\tau}{g(s-1)D_{50}}
```

Comment:
```suggestion
.. math::

	\theta = \frac{\tau}{g(s-1)D_{50}}
```

Reply:
You're also using tabs for indenting in a few places.  The document should not have any tabs, just spaces.

Reply:
Thanks. Is there any specific number of spaces I should place for the indenting format?

Reply:
I follow the python specification of 4 spaces but I don't think RST/sphinx cares.

Reply:
I looked into this further.  The preference is for 3 spaces: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#unordered-list-inside-ordered-list